### DISCUSSION TITLE
Delete branch with multi-gitter merge

### DISCUSSION THREAD

**Discussion by klieret** [lindell/multi-gitter#267] [Ideas]
Option to delete branches after merge.

**Comment by lindell** [comment]
Do you mean to disable the deletion of the branch after merge? To understand the usecase better, how do you intend to use it?

**Reply by klieret** [reply]
Oh, it _does_ delete it after merging? Sorry, then I was mistaken :)

**Reply by klieret** [reply]
In that case, sorry for the noise!

**Reply by lindell** [reply]
No problem :) Indeed the branch is always deleted on merge.
